profiles/ar71xx-generic.profiles

```diff
@@ -1,6 +1,7 @@
 archer-c7-v1
 archer-c7-v2
 archer-c7-v4
+archer-c7-v5
```

Comment:
ok, just seen this fixes 19d5f76a4711260622b1227879a0a70a4e4777f3, I'll squash both